### ISSUE TITLE
Messages Cut Off

### ISSUE THREAD

**Issue by apamment** [NuSkooler/enigma-bbs#73]
For :bug: bug reports, please fill out the information below plus any additional relevant information. If this is a feature request, feel free to clear the form.

**Short problem description**
If someone quotes the tear line, the rest of the message is cut off.

**Environment**
- [X] I am using Node.js v4.x or higher
- [X] `npm install` reports success
- Actual Node.js version (`node --version`):
4.4.7
- Operating system (`uname -a` on *nix systems):
Linux abigale 4.4.0-31-generic #50-Ubuntu SMP Wed Jul 13 00:07:12 UTC 2016 x86_64 x86_64 x86_64 GNU/Linux

**Expected behavior**
Display the full message

**Actual behavior**
When receiving a message from ftn networks where the sender has quoted the tear line, only the message before the tearline is persisted in the database.

**Steps to reproduce**
Send a message from another BBS where you quote the tear line /origin line.

**Comment by NuSkooler** [comment]
@apamment : Do you happen to have some packet / bundles you can provide I can use to debug this?

**Comment by apamment** [comment]
Sorry, I don't.

I just noticed it in fsxnet, been wondering for a while why people quoted a block of text and wrote nothing. Looking on the agency BBS at the same message shows that the tear line has been removed, but then follows the origin line followed by the rest of the message.

Also I noticed that in the message_meta table, the offending message does not have any FtnProperty Values and only MSGID and REPLY FtnKludge values.

I can email you my message database if you like?

**Comment by NuSkooler** [comment]
@apamment I'll have to get something to reproduce this with. You can tell your system to keep imported packets for a bit and I can test with them once you see it again. In `config.hjson`:

```
scannerTossers: {
  ftn_bso: {
   retainImportPacketPath: "/path/to/a/place/to/dump/pkt/files/"
  }
}
```

**Comment by apamment** [comment]
OK,
I sent a message from xibalba and quoted the tear/origin-line you can see on xibalba what the message should look like, and on happyland what it does look like when tossed. I've attached the packet.

PS. I had to zip it for github
[imported-2016-07-18T13.53.41.586-8c530e01.zip](https://github.com/NuSkooler/enigma-bbs/files/368311/imported-2016-07-18T13.53.41.586-8c530e01.zip)

**Comment by NuSkooler** [comment]
@apamment Perfect! This might have to wait till tomorrow but that should give me what I need, thanks!

**Comment by NuSkooler** [comment]
Should now be resolved with 09e399a060f921e3db60b2d3d7b2f9178ae701a6

TL;DR: Ensure Origin is at the start of the line & clean up goofy `endOfMessage` variable
